packages/keyboard/src/useCustomKeyboard.tsx

```diff
@@ -80,10 +76,10 @@ export function useCustomKeyboard(
                 setCustomKeyboard(undefined);
             }, duration);
         };
-        Keyboard.addListener('keyboardWillHide', callback);
+        const keyboardSubscription = Keyboard.addListener('keyboardWillHide', callback);
 
         return () => {
-            Keyboard.removeListener('keyboardWillHide', callback);
+            keyboardSubscription.remove();
```

Comment:
Thanks!

Reply:
Anyway it didn't help and I still see that annoying warning 🤬